src/test/java/com/ericsson/ei/subscriptionhandler/test/SubscriptionValidatorTest.java

```diff
@@ -356,28 +356,54 @@ public void validateRestPostMediaTypeInvalidMessage2Test() throws Exception {
      * - "kalle.kalle@domain.com"
      */
     @Test
-    public void validateNotificationMetaValidMetaTest() throws Exception {
+    public void validateNotificationMetaValidMailMetaTest() throws Exception {
         String notificationMeta = "kalle.kalle@domain.com";
+        String notificationType = "MAIL";
         try {
-            invokeMethod(subscriptionValidator, "validateNotificationMeta", notificationMeta);
+            invokeMethod(subscriptionValidator, "validateNotificationMeta", notificationMeta, notificationType);
         } catch (SubscriptionValidationException e) {
             assertTrue(e.getMessage(), false);
             return;
         }
     }
 
     @Test
-    public void validateNotificationMetaInvalidMetaTest() throws Exception {
+    public void validateNotificationMetaInvalidMailMetaTest() throws Exception {
         String notificationMeta = "kalle.kall  e@domain.com";
+        String notificationType = "MAIL";
+        try {
+            invokeMethod(subscriptionValidator, "validateNotificationMeta", notificationMeta, notificationType);
+        } catch (SubscriptionValidationException e) {
+            assertTrue(e.getMessage(), true);
+            return;
+        }
+        assertTrue(false);
+    }
+
+    @Test
+    public void validateNotificationMetaInvalidMetaTest() throws Exception {
+        String notificationMeta = "";
+        String notificationType = "REST_POST";
         try {
-            invokeMethod(subscriptionValidator, "validateNotificationMeta", notificationMeta);
+            invokeMethod(subscriptionValidator, "validateNotificationMeta", notificationMeta, notificationType);
         } catch (SubscriptionValidationException e) {
             assertTrue(e.getMessage(), true);
```

Comment:
please remove all assertTrue(...,True) in this file even if they are not written by you